### ISSUE TITLE
[BUG] GUIManager.BlockInput() does not block all input

### ISSUE THREAD

**Issue by SpikeHimself** [Valheim-Modding/Jotunn#407]
Details:
Jotunn Version: 2.12.6
Jotunn Submodule: GUIManager
Repeatability: 100%

**Problem Description:**
`BlockInput` does not block all input. Character movement is blocked correctly, but M still opens the map, and Esc opens the game menu.
See also: https://github.com/SpikeHimself/XPortal/issues/64

**Expected Behaviour:**
After calling `GUIManager.BlockInput(true)` all input should be blocked.

**Actual Behaviour:**
After calling `GUIManager.BlockInput(true)` some but not all input is blocked.

**Link to your server's and client's LogOutput.log**
N/A

**Comment by MSchmoecker** [comment]
Thanks for the report, this is fixed in 2.12.7.
Hopfully all edge cases are now catched again and input is blocked correctly.

**Comment by SpikeHimself** [comment]
@MSchmoecker Scrollwheel input is still not blocked in 2.12.7 (character zoom occurs). Do you prefer a new issue for this?

**Comment by MSchmoecker** [comment]
Was this a thing before the Hildir update? Anyway, it should be blocked too.
We can reopen this issue as it's about the same topic.

**Comment by SpikeHimself** [comment]
> Was this a thing before the Hildir update?

I'm honestly not sure.. I don't *think* so.

**Comment by tdrury** [comment]
I was the one the created the XPortal issue that prompted this issue. I can say specifically for XPortal, this was not an issue pre-Hildir. I play virtually every day and it was fine before Hildir and broken after. Of course, there were a lot of mods broken right after...

I've noticed with everything updated, mouse-wheel events still bleed through. When I've opened the XPortal dialog and I'm scrolling through all my portal names using the mouse wheel, it also causes the screen behind the dialog to zoom in/out. This has no adverse effect on the game, unlike the key presses bleeding through, but thought it might be useful to  relay this info on this thread.
